### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by Brooooooklyn**
Seems two tests in `checkbox.spec.ts` failed due to dependencies up to date, and it supposed to be fixed.

BTW. It's supposed to be no filter in `pull_request` config https://github.com/element-plus/element-plus/blob/dev/.github/workflows/yarn.js.yml#L9 nor the `CI` will not run when pull request from outside repo.

### CODE REVIEW

**Comment by Brooooooklyn** [comment]
@HerringtonDarkholme could you help me fix the issue in `checkbox.spec.ts` so we can merge this pull_request?

**Comment by HerringtonDarkholme** [comment]
I'm moving today and I will give it a look later. Thanks!

**Comment by HerringtonDarkholme** [comment]
I have fixed the test case. Please give it a look @JeremyWuuuuu @zouhangwithsweet

File: packages/checkbox/src/checkbox.vue [hunk 1]
```diff
@@ -176,16 +177,14 @@ export default defineComponent({
...
+    function handleChange(e: InputEvent) {
       if (isLimitExceeded.value) return
-      let value = ref(undefined)
-      if ((e.target as HTMLInputElement).checked) {
-        value.value = props.trueLabel === undefined ? true : props.trueLabel
-      } else {
-        value.value = props.falseLabel === undefined ? false : props.falseLabel
-      }
+      const target = e.target as HTMLInputElement
+      const value = target.checked
```
Comment (HerringtonDarkholme) [inline packages/checkbox/src/checkbox.vue:183]: Is there any reason that we allow `trueLabel` or `falseLabel` as `null`? Should we add a breaking change here? @JeremyWuuuuu @zouhangwithsweet @zazzaz

File: packages/checkbox/src/checkbox.vue [hunk 2]
```diff
@@ -111,19 +112,18 @@ export default defineComponent({
...
+    const selfModel = ref<unknown>(false)
     const focus = ref(false)
     const isLimitExceeded = ref(false)
     const store = computed(() => _checkboxGroup ? _checkboxGroup.modelValue.value : props.modelValue)
-    const model = computed({
+    const model = computed<unknown>({
       get() {
-        return isGroup.value ? store.value : props.modelValue !== undefined ? props.modelValue : selfModel
+        return isGroup.value
+          ? store.value : props.modelValue !== undefined ? props.modelValue : selfModel.value
```
Comment (HerringtonDarkholme) [inline packages/checkbox/src/checkbox.vue:122]: Note the `selfModel.value`

File: packages/checkbox/src/checkbox.vue [hunk 3]
```diff
@@ -139,12 +139,13 @@ export default defineComponent({
...
     const isChecked = computed(() => {
-      if (Object.prototype.toString.call(model.value) === '[object Boolean]') {
-        return model.value
-      } else if (Array.isArray(model.value)) {
-        return model.value.includes(props.label)
-      } else if (model.value !== null && model.value !== undefined) {
-        return model.value === props.trueLabel
+      const value = model.value
+      if (Object.prototype.toString.call(value) === '[object Boolean]') {
+        return Boolean(value)
```
Comment (HerringtonDarkholme) [inline packages/checkbox/src/checkbox.vue:144]: Note the return type of `isChecked`

File: packages/checkbox/src/checkbox.vue [hunk 4]
```diff
@@ -176,16 +177,14 @@ export default defineComponent({
       }
     }
 
-    function handleChange(e: UIEvent) {
+    function handleChange(e: InputEvent) {
       if (isLimitExceeded.value) return
-      let value = ref(undefined)
```
Comment (HerringtonDarkholme) [inline packages/checkbox/src/checkbox.vue:181]: Is there any reason we need to wrap the value in a `ref`? @zouhangwithsweet

**Review by jw-foss — APPROVED** [review]

File: packages/checkbox/src/checkbox.vue [hunk 5]
```diff
@@ -25,6 +25,7 @@
       <input
         v-if="trueLabel || falseLabel"
         v-model="model"
+        :checked="isChecked"
```
Comment (HerringtonDarkholme) [inline packages/checkbox/src/checkbox.vue:28]: Note the `checked` prop here. Vue rc 9 no longer treats non-boolean `model` as `true` in checkbox's v-model

**Review by zouhangwithsweet — APPROVED** [review]

**Pull request merged**